tests/Unit/Models/PageTest.php

```diff
@@ -77,4 +78,42 @@ public function page_url_will_adapt_to_changing_parent(): void
         $child->refresh();
         $this->assertSame($child->url, '/stuff/web-design');
     }
+
+    /** @test */
+    public function it_may_have_translations(): void
+    {
+        $page = Page::factory()->create();
+        $this->assertInstanceOf(Collection::class, $page->translations);
+    }
+
+    /** @test */
+    public function you_can_add_a_translation(): void
+    {
+        $pageEn = Page::factory()->create(['language' => 'en']);
+        $pageNl = Page::factory()->create(['language' => 'nl']);
+
+        $pageEn->translations()->attach($pageNl->id);
+
+        $pageEn->refresh();
+        $this->assertCount(1, $pageEn->translations);
+        $this->assertInstanceOf(Page::class, $pageEn->translations->first());
+    }
+
+    /** @test */
+    public function setting_a_translation_goes_both_ways(): void
+    {
+        $pageEn = Page::factory()->create(['language' => 'en']);
+        $pageNl = Page::factory()->create(['language' => 'nl']);
+
+        $pageEn->translations()->attach($pageNl);
+
+        $pageEn->refresh();
+        $pageNl->refresh();
+
+        $this->assertCount(1, $pageEn->translations);
+        $this->assertInstanceOf(Page::class, $pageEn->translations->first());
+
+        $this->assertCount(1, $pageNl->translations);
+        $this->assertInstanceOf(Page::class, $pageNl->translations->first());
```

Comment:
Volgens mij tekst je hiermee ook wat in `you_can_add_a_translation` wordt getest.

Reply:
Yes, inderdaad, ik breid die vorige test uit zodat de bidirectionaliteit daar ook getest wordt.